app/js/lib/server/network.js

```diff
@@ -66,8 +66,16 @@ export default class Network extends EventDispatcher {
    * @return {Promise}
    */
   fetchJSON(url, method = 'GET', body = undefined) {
-    return this[p.fetch](url, 'application/json', method, body)
-      .then((response) => response.json());
+    const jsonMime = 'application/json';
```

Comment:
nit: can you rename it to `accept` to reuse the parameter name from `this[p.fetch]`?